channelnotifier/channelnotifier.go

```diff
@@ -45,6 +46,13 @@ type ClosedChannelEvent struct {
 	CloseSummary *channeldb.ChannelCloseSummary
 }
 
+// PendingOpenChannelEvent represents a new event where a new channel has
+// entered a pending open state.
+type PendingOpenChannelEvent struct {
```

Comment:
nit: move up such that the channel states are presented in the natural order

Reply:
Makes sense, fixed.